src/material/schematics/ng-generate/table/files/__path__/__name@dasherize@if-flat__/__name@dasherize__-datasource.ts.template

```diff
@@ -54,46 +54,54 @@ export class <%= classify(name) %>DataSource extends DataSource<<%= classify(nam
    * @returns A stream of the items to be rendered.
    */
   connect(): Observable<<%= classify(name) %>Item[]> {
-    // Combine everything that affects the rendered data into one update
-    // stream for the data-table to consume.
-    const dataMutations = [
-      observableOf(this.data),
-      this.paginator.page,
-      this.sort.sortChange
-    ];
+     if (this.paginator && this.sort) {
+       // Combine everything that affects the rendered data into one update
+       // stream for the data-table to consume.
+       const dataMutations = [
```

Comment:
While we're at it, it might be better to pass these in as individual parameters to `merge`, rather than creating an array and then spreading it.